tests/arrayBuffer.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
meta doesn't need the closing `/` 
https://www.w3schools.com/tags/tag_meta.asp

Reply:
Hmmm, my linter added without me knowing. Do you specifically want it without?

Reply:
naw. it's okay if the linter's automatically adding it. maybe we can look at removing it later (for another PR)